test/test_crc32c.py

```diff
@@ -24,55 +24,57 @@
 import struct
 import unittest
 import warnings
+from typing import Any, Generator, NamedTuple, List
 
-from typing import Any, Generator
+import pytest
 
 try:
     import crc32c
-    sw_mode = os.environ.get('CRC32C_SW_MODE')
-    if sw_mode == 'none' and not crc32c.hardware_based:
+
+    sw_mode = os.environ.get("CRC32C_SW_MODE")
+    if sw_mode == "none" and not crc32c.hardware_based:
         raise RuntimeError('"none" should force hardware support')
-    elif sw_mode == 'force' and crc32c.hardware_based:
+    elif sw_mode == "force" and crc32c.hardware_based:
         raise RuntimeError('"force" should force software support')
 except ImportError:
     crc32c = None  # type: ignore[assignment]
 
 
 def ulonglong_as_bytes(x: int) -> bytes:
-    return struct.pack('<Q', x)
+    return struct.pack("<Q", x)
 
 
 def uint_as_bytes(x: int) -> bytes:
-    return struct.pack('<I', x)
+    return struct.pack("<I", x)
 
 
 def ushort_as_bytes(x: int) -> bytes:
-    return struct.pack('<H', x)
+    return struct.pack("<H", x)
 
 
 def uchar_as_bytes(c: int) -> bytes:
-    return struct.pack('<B', c)
+    return struct.pack("<B", c)
 
 
 def batched(x: bytes, size: int) -> Generator[bytes, None, None]:
     length = len(x)
     for i in range(0, length, size):
-        yield x[i: min(i + size, length)]
+        yield x[i : min(i + size, length)]
 
 
 def as_individual_bytes(x: bytes) -> Generator[bytes, None, None]:
     for b in x:
         yield bytes([b])
 
 
-@unittest.skipIf(crc32c is None, 'no crc32c support in this platform')
+@unittest.skipIf(crc32c is None, "no crc32c support in this platform")
 class TestMisc(unittest.TestCase):
 
     def test_zero(self) -> None:
-        self.assertEqual(0, crc32c.crc32c(b''))
+        self.assertEqual(0, crc32c.crc32c(b""))
 
     def test_keyword(self) -> None:
-        self.assertEqual(10, crc32c.crc32c(b'', value=10))
+        self.assertEqual(10, crc32c.crc32c(b"", value=10))
 
```

Comment:
These (and some slight other changes below) came from running `black` on this file. Consistent formatting & string delimiter usage is a big plus in my book and I feel little is gained by moving this into another PR.

Reply:
On my book too (I wasn't aware/used to black those days when I wrote the package the first time though, and I never came around to add it since it was mostly C code)